views/index.pug

```diff
@@ -120,8 +120,25 @@
               span.icon.is-small
                   i.fas.fa-sign-in-alt#auth-button-icon(aria-hidden='true' style='pointer-events: none;')
               span#sign-in-text Sign In
+          
+            .column#hamburger-social-icons.is-hidden(style='')
+              a(href='https://www.facebook.com/CuriousLearningGLP/' target="_blank")
+                span.icon.is-large
+                  i.fab.fa-facebook-square(style='font-size: 1.5rem; color: #707070; ')
+              a(href='https://twitter.com/curiouslearnglp' target="_blank")
+                span.icon.is-large
+                  i.fab.fa-twitter-square(style='font-size: 1.5rem; color: #707070; ')
+              a(href='https://www.instagram.com/curiouslearning_glp/' target="_blank")
+                span.icon.is-large
+                  i.fab.fa-instagram-square(style='font-size: 1.5rem; color: #707070; ')
+              a(href='https://www.linkedin.com/company/curious-learning' target="_blank")
+                span.icon.is-large
+                  i.fab.fa-linkedin(style='font-size: 1.5rem; color: #707070; ')
+              button.button#mailchimp-hamburger-button(style='margin: 0.4rem; text-decoration: none; background-color: rgba(0, 0, 0, 0); border: none')
+                span.icon.is-large
+                  i.far.fa-envelope(style='font-size: 1.5rem; color: #707070;')
 
-    .section#tabs-parent(style='z-index: 100; padding: 0;')
+    .section#tabs-parent(style='z-index: 100; padding-right: 0; padding-left: 0;')
```

Comment:
if we upgrade to the latest version of Bulma, this can be simplified to `.section.py-0#tabs-parent(style='z-index: 100;')`